src/views/todo/todo-list.ejs

```diff
@@ -2,4 +2,19 @@
 
 <h1>To Do List</h1>
 
+<form class="to-do-list" action="" method="PUT">
+
+  <label for="item">To Do:</label>
+  <input type="text" name="item" required>
+
+  <input type="submit" name="to-do-list-button" value="Submit">
+
+</form>
+
+<h2>List of To Dos:</h2>
+
+<ol>
+  <li> Example Text<br><input type="radio" name="to-do-list-item"> Complete</li>
```

Comment:
should this be a radio button or a check box? haven't seen the UI, just a thought.

Reply:
Complete 👍